addons/html_builder/static/tests/toolbox/widget.test.js

```diff
@@ -382,6 +390,80 @@ describe("WeNumberInput", () => {
         expect(".o-snippets-top-actions .fa-undo").toBeEnabled();
         expect(".o-snippets-top-actions .fa-repeat").not.toBeEnabled();
     });
+    test("hide/display base on applyTo", async () => {
```

Comment:
In order to make it easy to grep when a test fails it is better to have unique strings that describe the tests.
Can you add an identifier to make then unique (eg `test("hide/display base on applyTo (WeNumberInput)", async () => {`)